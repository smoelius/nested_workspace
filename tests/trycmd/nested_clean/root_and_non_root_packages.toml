args = ["nested", "clean"]

[bin]
name = "cargo-nested"

[fs]
cwd = "../../../fixtures/root_and_non_root_packages"
