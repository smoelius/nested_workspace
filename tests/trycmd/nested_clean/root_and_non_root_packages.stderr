     Removed [..] files[..]
<<< [CWD]/root_package
     Removed [..] files[..]
>>> [CWD]/root_package
